File: AmongNeewAmigosFrontEnd/AmongNeewAmigosFrontEnd/src/app/pages/dtos/PollAnalyseDetails.ts
export class PollAnalyseDetails {
    constructor() {
    }

    resultsWithoutIpFilter: Map<string,number>;
    resultsWithIpFilter: Map<string,number>;
    voteTitle: string;
    userVotes: Array<string>;
    totalVotesWithoutIpFilter: number;
    totalVotesWithIpFilter: number;
}

File: AmongNeewAmigosBackEnd/src/main/java/leandromangieri/portfolio/core/dto/PollAnalyseDetails.java
package leandromangieri.portfolio.core.dto;

import lombok.Data;

import java.util.List;
import java.util.Map;

@Data
public class PollAnalyseDetails {
    private Map<String, Integer> resultsWithoutIpFilter;
    private int totalVotesWithoutIpFilter;

    private Map<String, Integer> resultsWithIpFilter;
    private int totalVotesWithIpFilter;

    private String voteTitle;
    private List<String> userVotes;
}
